src/components/Nav/NavHeader.js

```diff
@@ -1,10 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link, useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
-import { Lock, UserPlus, User, Headphones } from 'react-feather';
+import { Lock, Unlock, UserPlus, User, Headphones } from 'react-feather';
 
 function NavHeader() {
+  const [currentLogin, setCurrentLogin] = useState(false);
   const navigate = useNavigate();
+  const isLogin = localStorage.getItem('TOKEN');
+
+  const LogoutHandler = () => {
+    localStorage.removeItem('TOKEN');
+    setCurrentLogin(!currentLogin);
```

Comment:
```suggestion
    setCurrentLogin(prev=>!prev);
```
함수형 업데이트로 수정해보시면 어떨까요

Reply:
효민님 리뷰 반영이 안된것같아요 한번 확인 해 보시겠습니까???